ansible/roles/aws/tasks/delete.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
These tasks have been linked back to a playbook at this time yet right?  If not could we put them in a separate p/r when that is ready?